include/proxy-wasm/wasm_vm.h

```diff
@@ -124,6 +124,17 @@ struct WasmVmIntegration {
                                  void *ptr_to_function_return) = 0;
 };
 
+enum class FailState : int {
+  NotFailed = 0,
```

Comment:
nit: Success is easier to grok than double negatives.

Reply:
changed to Ok